epcis/src/main/java/io/openepcis/model/epcis/modifier/CustomInstantAdapter.java

```diff
@@ -20,6 +20,7 @@
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
+import java.time.temporal.ChronoUnit;
 
 public class CustomInstantAdapter extends XmlAdapter<String, OffsetDateTime> {
 
```

Comment:
missed out on marshal method

Reply:
I have added truncate to millis in the marshal method also.